packages/eslint-config-react-app/index.js

```diff
@@ -78,7 +78,13 @@ module.exports = {
         'no-undef': 'off',
 
         // Add TypeScript specific rules (and turn off ESLint equivalents)
-        '@typescript-eslint/no-angle-bracket-type-assertion': 'warn',
+        '@typescript-eslint/consistent-type-assertions': [
```

Comment:
We should be able to set this to `'warn'` without specifying options as they appear to be the default: https://github.com/typescript-eslint/typescript-eslint/blob/v2.0.0/packages/eslint-plugin/docs/rules/consistent-type-assertions.md#options.